src/webview/serverless-function/app/createFunction.tsx

```diff
@@ -0,0 +1,327 @@
+    }
+
+    selectFolder = (): void => {
+        VSCodeMessage.postMessage({
+            action: 'selectFolder',
+            noWSFolder: true,
+            name: this.state.functionData.name
+        });
+    }
+
+    handleWsFolderDropDownChange = (_e: any, value: Uri | string): void => {
+        console.log('VAlue:::', value);
+        if (typeof value === 'string' && value === 'New Folder') {
+            VSCodeMessage.postMessage({
+                action: 'selectFolder',
+                noWSFolder: true
+            });
+        } else if (typeof value === 'object') {
+            this.setState({
+                wsFolderPath: value
+            });
+        }
+    }
+
+    convert = (value: string): string => {
+        return value.replace('/\s+/g', '').toLowerCase();
+    }
+
+    handleDropDownChange = (_event: any, value: string, isLang = false): void => {
+        if (isLang) {
+            if (value !== 'Python') {
+                this.setState({
+                    templates: ['Cloud Events', 'HTTP']
+                });
+            } else {
+                this.setState({
+                    templates: ['Cloud Events', 'Flask', 'HTTP', 'WSGI']
+                })
+            }
+            this.setState({
+                language: value,
+                template: ''
+            });
+        } else {
+            this.setState({
+                template: value
+            });
+        }
+    }
+
+    handleCreateBtnDisable(): boolean {
+        return !this.state.functionData || this.state.functionData.name.length === 0 || this.state.functionData.error
+            || !this.state.imageData || this.state.imageData.name.length === 0 || this.state.imageData.error
+            || !this.state.wsFolderPath || this.state.language.length === 0 || this.state.template.length === 0;
+    }
+
+    createFunction = (): void => {
+        this.props.onCreateSubmit(this.state.functionData.name,
+            this.convert(this.state.language), this.convert(this.state.template), this.state.wsFolderPath, this.state.imageData.name)
+    }
+
+    render(): React.ReactNode {
+        const { wsFolderItems, wsFolderPath, functionData, templates, language, template, images, imageData } = this.state;
+        const languages = ['Go', 'Node', 'Python', 'Quarkus', 'Rust', 'Spring Boot', 'TypeScript'];
+        const folders: any[] = ['New Folder'];
+        folders.push(...wsFolderItems);
+        const filter = createFilterOptions<string>();
+        const imageRegex = RegExp('[^/]+\\.[^/.]+\\/([^/.]+)(?:\\/[\\w\\s._-]*([\\w\\s._-]))*(?::[a-z0-9\\.-]+)?$');
+        return (
+            <>
+                <FormControl sx={{ margin: '2rem 0 0 2rem', flexFlow: 'row', width: 650, gap: 0.2 }}>
+                    <FormControl sx={{ width: 180 }}>
+                        <Button variant='contained'
+                            disabled={true}
+                            className='labelStyle'>
+                            Function Name*
```

Comment:
Let's keep the star with space between the text. And keep it consistent across Labels.